tensorflow/compiler/tf2tensorrt/kernels/trt_engine_resource_ops.cc

```diff
@@ -211,6 +211,17 @@ class SerializeTRTResource : public OpKernel {
 
     int num_serialized_engines = 0;
     if (save_gpu_specific_engines_) {
+      // If user requests TRT engines export, recursively create
+      // requisite directories.
+      bool export_trt_engines = false;
```

Comment:
A better approach might be:
`const bool export_trt_engines = getenv("TF_TRT_EXPORT_TRT_ENGINES_PATH") != nullptr;`, and then doing `if (export_trt_engines)` in the next line.

Reply:
or may be use const char* to replace the const bool:
const char * export_trt_engine_env = getenv("...");
if (export_trt_engine_env) {
...}

Reply:
Or what Bixia said, provided we don't keep `export_trt_engine` anymore!

Reply:
@bixia1 @pjannaty  -- ack...new rev coming up!